gef-ui/server/index.js

```diff
@@ -20,49 +19,54 @@ const app = express();
 
 const PORT = process.env.PORT || 5006;
 
-// Fail-safe fallback to a 256-bit random value:
-const SESSION_SECRET = process.env.SESSION_SECRET || crypto.randomBytes(256 / 8).toString('hex');
+if (!process.env.SESSION_SECRET) {
+  console.error('GEF UI server - SESSION_SECRET missing');
+}
+
+if (!process.env.REDIS_HOSTNAME) {
```

Comment:
We also get a connection error on line 53 is REDIS_HOSTNAME is invalid. Do we need this too?